src/pykcs11.i

```diff
@@ -1023,6 +1082,9 @@ typedef unsigned long CK_RV;
 #define CKM_HKDF_DATA                  0x0000402b
 #define CKM_HKDF_KEY_GEN               0x0000402c
 #define CKM_VENDOR_DEFINED             0x80000000UL
+#define CKM_DES2_DUKPT_MAC             (CKM_VENDOR_DEFINED + 0x612)
```

Comment:
I can't include vendor specific mechanisms.

Are these mechanisms planned for a future version of PKCS#11 standard?